paymentsheet/src/main/java/com/stripe/android/common/ui/BottomSheet.kt

```diff
@@ -174,3 +184,16 @@ private suspend fun repeatUntilSucceededOrLimit(
         }
     }
 }
+
+private val isRunningTest: Boolean
+    get() {
+        val isUnitTest = runCatching {
```

Comment:
Can we early exit if BuildConfig.Debug is false?

Reply:
Added!